queries.sql

```diff
@@ -7,4 +7,45 @@ SELECT date_of_birth FROM animals WHERE name IN ('Agumon','Pikachu');
 SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered;
 SELECT * FROM animals WHERE name <> 'Gabumon';
-SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+ROLLBACK;
+
+
+BEGIN;
+UPDATE animals 
+SET species = 'digimon'
+WHERE name LIKE '%mon';
+UPDATE animals 
+SET species = 'pokemon'
+WHERE species IS NULL;
```

Comment:
- [ ] Hello! 🚀 After updating the animal species to "Digimon" and "Pokemon" respectively, in accordance with the specified conditions, please take a moment to verify that the changes have been successfully made in the database before committing. 🔍🔧 Once committed, double-check to ensure that the changes have indeed persisted. 📊💻 Your attention to this verification process is greatly appreciated! 👏

```suggestion
   BEGIN;
   UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon'; 
   UPDATE animals SET species = 'pokemon' WHERE species IS NULL; 

   SELECT species from animals; -- verify that change was made
   COMMIT;
   SELECT species from animals; -- verify that change persists after commit

```